src/auth/routes.rs

```diff
@@ -60,9 +93,13 @@ pub async fn handle_auth(
     if req.path().eq("/auth/login/") {
         return login(req, edition).await;
     } else if req.path().eq(fpm::auth::github::ACCESS_URL) {
```

Comment:
use a `match` statement, instead of `if and else if`.

Reply:
done